docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -527,7 +527,11 @@ as follows:
 
 **Birth Weight**
 
-The effect of maternal consumption of iron fortified food on infant birth weight was obtained directly from Haider et al. (2013). According to this data source, mean birth weight among babies born to mothers who consumed iron fortified foods had a mean birth weight **15.1 grams (95% CI: 6.0, 24.2)** higher than babies born to mothers who did not consume iron fortified foods. 
+The effect of maternal consumption of iron fortified food on infant birth 
+weight was obtained directly from Haider et al. (2013). According to this data 
+source, mean birth weight among babies born to mothers who consumed iron 
+fortified foods had a mean birth weight **15.1 grams (95% CI: 6.0, 24.2)** 
```

Comment:
I thought this effect was for a specific amount of iron supplement (10g?) and we were planning to use a linear dose-response relationship to figure out some X*15.1 g for the effect of iron fortification.

Reply:
Yes, I was unaware about this at first, but I just pushed another commit that proposes an approach to this in the PR here: https://github.com/ihmeuw/vivarium_research/pull/179